### DISCUSSION TITLE
Bug: puppet-agent 8.2.0 on gentoo not detecting active init

### DISCUSSION THREAD

**Discussion by samuraiii** [puppetlabs/community#51] [General]
Hi all,
there is a regression for puppet-agent on gentoo linux, presumably between versions >8 and 8<=.
The 7.X version was detecting two inits on my box but was choosing the right every time (systemd).
Version 8.2.0 detects also both inits but chooses the "init" one, which breaks all systemd related facts.

I am not sure where to post this since tickets are obsolete, so I am sorry for possible misposting.
Cheers
S

**Comment by binford2k** [comment]
Hi. I've moved this report to https://github.com/puppetlabs/puppet-agent/issues/2444
